crates/gpui/src/platform.rs

```diff
@@ -784,7 +784,7 @@ pub struct TitlebarOptions {
     /// The initial title of the window
     pub title: Option<SharedString>,
 
-    /// Whether the titlebar should appear transparent (macOS only)
+    /// Whether the titlebar should appear transparent (macOS and Windows only)
```

Comment:
```suggestion
    /// Whether the titlebar should appear transparent (macOS only)
    /// Should the default system title bar be hidden to allow for a custom-drawn title bar? (Windows and Linux)
```

Reply:
Hmm, I though the purpose was the same for Mac as for Windows/Linux. But I think that your comment states the intent more clearly 🙂
I updated the PR with it now.